data/mllib/sample_hierarchical_data.csv

```diff
@@ -0,0 +1,150 @@
+5.1,3.5,1.4,0.2
```

Comment:
Good point =) Leave as is then. Maybe at some point we should give all the vector-valued example data sets the same format / file type just for consistency, but that can be a separate PR.